uxarray/grid/grid.py

```diff
@@ -904,6 +905,18 @@ def face_jacobian(self):
             _ = self.face_areas
         return self._face_jacobian
 
+    @property
+    def index_holes(self):
+        """Check the grid for faces that are next a hole inside a partial grid.
+
+        Returns
+        """
+        if "index_holes" not in self._ds:
+            self._ds["index_holes"] = _index_faces_on_partial_grid(
+                self.edge_face_connectivity.values
+            )
+        return self._ds["index_holes"]
+
```

Comment:
I like this implementation a lot, but I think we can maybe use a better name for this. For reference, we have the following which is the indices of all the faces that cross the antemeridian. 

https://github.com/UXARRAY/uxarray/blob/778e05e27af90a27e4ea81d84e137a33b6064a7a/uxarray/grid/grid.py#L846-L851

I think `hole_edge_indices` would be appropriate. We should also explicitly define a hole as a region of the grid that is not covered by any geometry. 

I'd also suggestion modifying the docstring to describe what variable is stored within the property, like we've done with the others.

Reply:
Where do you think an appropriate place to construct this would be? I don't think validation is a good place for it anymore. Connectivity doesn't really make that much sense either. Geometry maybe?

Reply:
> Where do you think an appropriate place to construct this would be? I don't think validation is a good place for it anymore. Connectivity doesn't really make that much sense either. Geometry maybe?

+1 for geometry